mirrord/operator/src/crd.rs

```diff
@@ -3,7 +3,7 @@ use std::{
     fmt::{Display, Formatter},
 };
 
-use kube::CustomResource;
+use kube::{CustomResource, Resource};
```

Comment:
You need to undo this for the docs check not to return an error.

Reply:
Need this for the `meta()` calls. Removed the `kube::Resource` part from the docs instead.